plugins/inputs/ravendb/README.md

```diff
@@ -0,0 +1,216 @@
+# RavenDB Input Plugin
+
+Reads metrics from RavenDB servers via monitoring endpoints APIs.
+
+Requires RavenDB Server 5.2+.
+
+### Configuration
+
+The following is an example config for RavenDB. **Note:** The client certificate used should have `Operator` permissions on the cluster.
+
+```toml
+[[inputs.ravendb]]
+  ## Node URL and port that RavenDB is listening on.
+  url = "https://localhost:8080"
+
+  ## RavenDB X509 client certificate setup
+  tls_cert = "/etc/telegraf/raven.crt"
+  tls_key = "/etc/telegraf/raven.key"
+
+  ## Optional request timeout
+  ##
+  ## Timeout, specifies the amount of time to wait
+  ## for a server's response headers after fully writing the request and 
+  ## time limit for requests made by this client.
+  # timeout = "5s"
+
+  ## List of statistics which are collected
+  # At least one is required
+  # Allowed values: server, databases, indexes, collections
+  #  
+  # stats_include = ["server", "databases", "indexes", "collections"]
+
+  ## List of db where database stats are collected
+  ## If empty, all db are concerned
+  # db_stats_dbs = []
+
+  ## List of db where index status are collected
+  ## If empty, all indexes from all db are concerned
+  # index_stats_dbs = []
+  
+  ## List of db where collection status are collected
+  ## If empty, all collections from all db are concerned
+  # collection_stats_dbs = []
+```
```

Comment:
small nitpick: some of the comments on these settings end with periods and some don't, likely should be consistent

Reply:
removed dots